src/components/chip-set/chip-set.e2e.ts

```diff
@@ -291,6 +293,16 @@ describe('limel-chip-set', () => {
                     removable: true,
                 });
             });
+
+            // This test doesn't work at the moment, and I can't run the e2e
+            // tests locally to debug it. So I'm skipping it for now. /Ads
+            it.skip('attaches the chip to the click event', () => {
```

Comment:
Oh ffs… it seems like this isn't possible to test, because the "event" object you get from the spy isn't actually the real event object which we added the `Lime` property to… 😣

Reply:
Got an idea, and it worked! Pushing a fixup in a moment!